config/settings/local-sample.py

```diff
@@ -1,28 +1,53 @@
+import sentry_sdk
 from .base import * # pylint: disable=unused-wildcard-import
 
 DEBUG = False
 ALLOWED_HOSTS = [
-    'jepostule.pole-emploi.fr',
-    'localhost',
+    'jepostule.labonneboite.pole-emploi.fr',
 ]
-STATIC_ROOT = '/var/www/jepostule/static'
+STATIC_ROOT = 'ChangeMe'
 
-# SECRET_KEY = 'setme'
-DATABASES['default']['HOST'] = 'postgres'
-# DATABASES['default']['PASSWORD'] = 'setme'
+SECRET_KEY = 'ChangeMe'
+DATABASES['default']['HOST'] = 'ChangeMe'
+DATABASES['default']['PASSWORD'] = 'ChangeMe'
 
-REDIS_HOST = 'redis'
+REDIS_HOST = 'ChangeMe'
 
-KAFKA_BOOTSTRAP_SERVERS = ['kafka:9092']
+KAFKA_BOOTSTRAP_SERVERS = ['ChangeMe']
 
-# EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
-# EMAIL_HOST = 'setme'
-# EMAIL_PORT = 0
-# EMAIL_HOST_USER = 'setme'
-# EMAIL_HOST_PASSWORD = 'setme'
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'ChangeMe'
+EMAIL_PORT = 00
+EMAIL_HOST_USER = 'ChangeMe'
+EMAIL_HOST_PASSWORD = 'ChangeMe'
 
-# import sentry_sdk
-# sentry_sdk.init(
-    # "https://set@sentry.io/me",
-    # environment="production"
-# )
+LOGGING['handlers']['fluentbit'] = {
+    'class': 'logging.handlers.SysLogHandler',
+    'formatter': 'simple',
+    'address': ('fluentbit', 000),
+}
+LOGGING['handlers']['file'] = {
+    'class': 'logging.handlers.WatchedFileHandler',
+    'formatter': 'simple',
+    'filename': 'ChangeMe.log',
+}
+LOGGING['loggers']['jepostule']['handlers'] += ['fluentbit', 'file']
+
+JEPOSTULE_BASE_URL = 'https://jepostule.labonneboite.pole-emploi.fr'
+JEPOSTULE_NO_REPLY = 'change@me.com'
+EVENT_CALLBACK_SECRET = 'ChangeMe'
+EMAIL_DELIVERY_SERVICE = 'mailjet'
+MAILJET_API_KEY = "ChangeMe"
+MAILJET_API_SECRET = "ChangeMe"
+
+GOOGLE_ANALYTICS_TRACKING_ID = "ChangeMe"
+HOTJAR_SITE_ID = 123456789
+
+sentry_sdk.init(
+    "https://change_me@sentry.io/ChangeMe",
+    environment="production"
+)
+
+MEMO_API_URL = 'https://memo.pole-emploi.fr/rest/api/v1'
+MEMO_API_SECRET = 'ChangeMe'
+MEMO_API_VERIFY_SSL = True
```

Comment:
mmmmm je suis un peu perplexe devant tous ces changements.

Le but de ce sample est de fonctionner out of the box (ou quasi) pour un dev externe qui veut faire tourner le projet en local. L'ancienne version jouait bien ce rôle, mon local dev tournait avec. J'avais juste rajouté un peu de conf d'API MEMO quand je travaillais spécifiquement sur le sujet.

J'ai l'impression que tu as changé pas mal de valeurs qui faisent fonctionner la plomberie en local dev par un `ChangeMe`, du coup rajoutant des étapes au dev externe pour faire tourner son local dev.

Exemples:
`DATABASES['default']['HOST']` pointait vers le bon nom du container postgre, idem pour `REDIS_HOST`. Etc etc.

Je pose ma question autrement: mets toi dans la peau d'un dev externe qui installe en local sans pouvoir/vouloir nous contacter. Laquelle de l'ancienne et de la nouvelle version de ce sample lui facilitent le plus le boulot i.e. fonctionne "out of the box" avec le moins d'efforts?

Reply:
Je reconnais que je contredis peut-être un peu ce que j'avais dit auparavant. Sorry!

Reply:
Oui, je suis d'accord avec toi. Mais cela vient tout simplement du fait que j'ai cru que `local.py` ne servait qu'en prod et pas en dev ! :-)
Après notre discussion de ce matin, je regarde pourquoi c'est `local.py` qui est pris en considération en dev.

Reply:
Après quelques investigations, je suis moi aussi perplexe mais plus sur l'utilisation du `local.py`.

Quand je lance `make run` après un `make services`, c'est bien `config.settings.base` qui est utilisé : 

```
$ make run
./manage.py runserver
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
August 09, 2019 - 17:15:54
Django version 2.2.2, using settings 'config.settings.base'
Starting development server at http://127.0.0.1:8000/
Quit the server with CONTROL-C.

```

Si par contre je fais `make platform` (qui ne fonctionne pas, mais pour d'autres raisons), c'est bien le fichier de config du Dockerfile qui est pris en compte : `local.py`.

C'est très perturbant, je trouve, d'avoir deux fichiers de config potentiels en dev. Je serais d'avis de clarifier tout ça mais j'ai l'impression de passer à coté de quelque chose car, toi, tu as bien `local.py` qui est utilisé en lançant `make run`. Si tu avais deux minutes pour en parler @vgrange ce serait chouette ! :)
Merci !